### ISSUE TITLE
Tableview can't change rowheight

### ISSUE THREAD

**Issue by sciolism-9** [israel-dryer/ttkbootstrap#198]
Tableview can't change rowheight

example:
```python
import ttkbootstrap as ttk
from ttkbootstrap.tableview import Tableview
from ttkbootstrap.constants import *

root = ttk.Window()
root.geometry('600x600')

style = ttk.Style()
style.configure("Treeview.Heading", font=(None, 15), rowheight=40)
style.configure("Treeview", font=(None, 15), rowheight=40)
colors = root.style.colors

heading = ['app', 'version']
data = [['app1', '1.0'], ['app2', '1.1'],['app3', '1.2']]

tv = Tableview(
            master=root,
            coldata=heading,
            rowdata=data,
            paginated=True,
            searchable=True,
            autofit=True,
            bootstyle=PRIMARY,
            stripecolor=(colors.light, None),
        )

tv.pack(fill=BOTH, expand=YES, padx=1, pady=1)

if __name__ == '__main__':
    root.mainloop()
```

I found the fontsize is work,but the rowheight is not.

**Comment by israel-dryer** [comment]
@sciolism-9, Ok, I just realized what was happening. This issue was actually fixed in #99.

The actual base ttk style that is applied to the `Tableview` widget is **Table.Treeview**.

In this situation, you are applying the **PRIMARY** bootstyle, which subclasses the style so that it is now **primary.Table.Treeview**. This is the style you must configure for the changes to take place.

The reason that the font is changing but the rowheight is not in your original example is because the  Tableview itself is a subclass of the Treeview and the font _is not set by default_ in any of those styles, so your config basically becomes the default.

```python
style.configure("primary.Table.Treeview", font=(None, 15), rowheight=40)
style.configure("primary.Table.Treeview.Heading", font=(None, 15))
```

**Comment by israel-dryer** [comment]
> Thanks.I had checked the #99 issue, but I don't get the key.

Were you able to get it to work in your code?
